lib/config/options/index.ts

```diff
@@ -2416,17 +2416,18 @@ const options: RenovateOptions[] = [
   },
   {
     name: 'concurrentRequestLimit',
-    description: 'Limit concurrent requests per host.',
+    description:
+      'Limit concurrent requests per host. Set to `-1` for no limit. Default value is 5 concurrent requests.',
     type: 'integer',
     stage: 'repository',
     parents: ['hostRules'],
-    default: null,
```

Comment:
Would it help for docs readability if this default was set to 5 here, even if it's not directly used?